interfaces/builtin/all.go

```diff
@@ -27,6 +27,15 @@ import (
 	"github.com/snapcore/snapd/snap"
 )
 
+// BuiltinIfaceCount represents the total number of built-in interfaces.
+// It should only be used to detect newly added interfaces and remind
+// developers to create documentation available at the default doc URL.
+const BuiltinIfaceCount = 190
```

Comment:
I don't think tying this to the task of creating docs is a workable approach.  Can't we have a InterfaceCount function that returns len(allInterfaces) ?

we need to do something else about the docs, we could have a nightly test that checks the urls?